### ISSUE TITLE
[SUGGESTION]: Pokémon 3: The Movie Event

### ISSUE THREAD

**Issue by Shinesprite41** [pokeclicker/pokeclicker#4800]
### Current Behavior

N/A

### Improved Behavior

An event should be added based on Pokemon 3: The Movie - Spell of the Unown: Entei, similar to the Mewtwo Strikes Back event. In the movie, Entei, who is created by a group of Unowns for the sake of keeping this girl named Molly safe, kidnaps Ash's mother and tries to stop Ash & the gang from rescuing her. There's a much longer synopsis available online, but that's the main plot point.

### Reason for change

Of course, the first reason is because people enjoy events, and I believe that if we have the Mewtwo event, we should look into having events for the other movies as well. The amount of events overall seems a bit underwhelming, and, while I'm not suggesting this in this post, I could see a "movie month" where the events overlap (still limiting them to a week or so each, and staggering when they start). Secondly, Johto is mildly underwhelming, with all of its events being mixed-regions (pokemons/quests appear in Kanto, Johto, and Hoenn) aside from the Christmas event with Santa Celebi (even then, Santa Snorlax appears in all regions). Lastly, I think it gives the opportunity for players to stray away from the linearity that Johto has and provides more value inside of Johto, which does also host an underwhelming number of Pokemon, being the third lowest next to Kalos and Sinnoh. 

### **Where**
The event is accessible from Goldenrod onwards (could make it later, but I believe canonically this is around when the movie takes place).

### **When**
Based on the Mewtwo Strikes Back event, I think this one should start either immediately after or a few days before the end of the Mewtwo event.

### **Event Layout**
This is obviously adjustable as the person who creates the event sees fit, but this is a layout of how I imagine the event would work. Do note that it is obviously much longer than the Mewtwo event, which is very short overall, but I believe that event should be reworked as well (and will make another post about that in the future) to be much longer and engaging. 
When the event starts, a new town emerges on the Johto map called Greenfield. This can have a route connected via Route 34 heading East, which is where Greenfield would be, roughly. This route can have the same Pokemon that appear on Route 34. Once clearing the Route, a Pokemon trainer named Lisa shows up, and you must defeat her to access the town. In the movie, Lisa's team was: Aipom, Granbull, Girafarig, Butterfree, Mankey, and Quagsire. Once you defeat her, you can access Greenfield, where you have to follow a plot dialogue similar to the movie with the town being Crystallized and the TV reporter discussing if it has anything to do with Dr. Hale disappearing and Professor Oak and Delia Ketchum arriving to the scene. After the report, there should be an Entei boss fight. Once the boss fight is over, Entei kidnaps Delia Ketchum and runs away. 
From here, the player enters a dungeon inside of an overworld mansion sprite. This mansion will likely end up having 2-3 dungeons inside of it and can consist of similar mons to the route before-hand, but should also feature Unown (in increasing amounts the further into the dungeons you are, so the first dungeon might only have 1-2 unown types, the second has 4, and the 3rd is only Unown. The boss of the first two dungeons would be the girl who the Unown are listening to in the movie, Molly, who has a team of Flaaffy, Teddiursa, and Phanpy in the first Dungeon and Kingdra and Mantine in the second (which follows the movie of having Brock fight the first team and then Misty fight the second). The third dungeon has a Boss Entei fight. Upon defeating the Entei, more dialogue happens to finish out the movie events, and the dungeons are replayable for the remainder of the event

### **Featured Pokemon**
During this event, there should be a couple new Pokemon available. Firstly, the Entei, while not having a distinct difference in the movie, could be a semi-crystallized form, or have a psychic aura surrounding it to signify the Unowns are creating it. Secondly, there is a subset of Unowns visible in the Ruins in the movie, which can become Unowns in the dungeons, attached here.
![500px-Composite_Unreleased_Unown](https://github.com/pokeclicker/pokeclicker/assets/73353569/bde90c00-8a5e-4664-8eec-0dcfbed6b663)

**Comment by Shinesprite41** [comment]
This is the best list I could make for the real-world equivalents to the Unown

In order of the photos from left to right, top to bottom. 
1) Я (Cyrillic "Ya")
2) Ҁ (Cyrillic "Koppa")
3) Ю (Cyrillic "Yu")
4) 𐒈 (Osmanya "Sa")
5a) 𐭧 (Pahlavi "Heth")
6) ω (Greek "Omega")
7) 𑫏 (Pau Cin Hau “Tha”) 
8) ᤍ (Limbu "Da")
9) Я (Cyrillic "Ya") again
10) Ⰾ (Glagolitic "Ljudije")
11) Σ (Greek “Sigma”)
12) б (Cyrillic "Be")
13a) ﻉ  (Arabic "Ain")



I also wish to rework the Mewtwo movie event to be a storyline event and potentially add a couple other movies as events, but I wish to see how well this one is liked / responded to by the coding team, first

**Comment by RealPendragon** [comment]
The Mewtwo Event was just recently reworked.

**Comment by Shinesprite41** [comment]
Yes, but from what I've seen of it, it is very basic still and doesn't follow a storyline or anything remotely similar. I could be wrong, as I don't remember from when it was active, but I did look for it in the repository and didn't find much interesting. I think it could be a much more involved event that is actually interesting, rather than just having a single Mewtwo dungeon and having the roamers chilling around Kanto.

**Comment by DapperDwarf** [comment]
so why would it be an event rather than just a questline at that point?  I always thought events were simpler on purpose because they're limited time

**Comment by Shinesprite41** [comment]
It would be changed to be a storyline event because it would be a unique and fun interaction rather than a very boring singular dungeon. If an event is going to be based off of a movie (in Mewtwo’s case, be as a celebration of the anniversary of the movie), it should be more involved. It’s okay to have events with a storyline. The current togepi event has a mini storyline, for example. Along with this, the storyline events still wouldn’t be ridiculously long. Even with the storyline I suggested for the Entei

**Comment by Shinesprite41** [comment]
The close was a fat finger moment, my bad
